src/users/controllers/users.controller.ts

```diff
@@ -47,13 +47,13 @@ export class UsersController {
 
   @ApiDeleteUser()
   @Delete(':id')
-  remove(@Param('id', PositiveIntPipe) userId: number) {
-    this.usersService.deleteUser(userId);
+  async remove(@Param('id', PositiveIntPipe) userId: number) {
+    await this.usersService.deleteUser(userId);
     return { statusCode: 204, message: 'No Content' };
   }
```

Comment:
```suggestion
  @HttpCode(HttpStatus.NO_CONTENT)
  remove(@Param('id', PositiveIntPipe) userId: number) {
    return this.usersService.deleteUser(userId);
  }
```
이렇게 해주면 더 깔끔해 보일 것 같은데 어떻게 생각하시나요? 
HttpCode 데코레이터에 204를 넣어주면 자동으로 response body에 아무 값도 들어 있지 않도록 처리 됩니다.

그리고 실제 Response로 가는 status code는 200이고 return 값에서만 204라고 되어 있네요.
추가적으로 204는 정말 아무 값도 response body에 넣지 않아야 하는데 return 값에 204, message를 명시해주는 것 자체가 좀 이상한 것 같습니다

Reply:
맞습니다. 저도 이렇게 바꿔줄까 고민했습니다만, 제 작업 코드가 아니라 수정을 최대한 절제 했습니다.
좋은 지적 감사하고 다른 팀원 분들이 확인 해보도록 말씀드리겠습니다.